python/tank/authentication/sso_saml2/utils.py

```diff
@@ -77,6 +80,7 @@ def _get_site_infos(url, http_proxy=None):
         else:
             get_logger().info("Infos for site '%s' found in cache", url)
             infos = INFOS_CACHE[url][1]
+    # pylint: disable=broad-except
```

Comment:
I 'm surprised you're getting this warning. Did you use `flake8`? Your PR seems to imply you did, but `except Exception as e` is not considered broad usually...

Reply:
Pylint is complaining, not flake8.